README.md

```diff
@@ -16,7 +16,7 @@
 # 📣 What does this dbt package do?
 - Materializes [Amazon_ads staging tables](https://fivetran.github.io/dbt_amazon_ads_source/#!/overview/amazon_ads_source/models/?g_v=1&g_e=seeds), which leverage data in the format described by [this ERD](https://fivetran.com/docs/applications/amazon-ads#schemainformation). These staging tables clean, test, and prepare your Amazon Ads data from [Fivetran's connector](https://fivetran.com/docs/applications/amazon-ads) for analysis by doing the following:
   - Names columns for consistency across all packages and for easier analysis
-  - Removes any rows that are soft-deleted
+  - Removes any rows that are soft-deleted 
```

Comment:
I don't think we will end up doing this in the package. Since there were no `_fivetran_deleted` columns in the source, we can assume this step will not be required.

Reply:
Done.